index.d.ts

```diff
@@ -23,6 +23,7 @@ export { ITestReporter } from "./lib/interfaces/test-reporter";
 export { screencapture } from "./lib/helpers/screenshot-manager";
 export { LogImageType } from "./lib/enums/log-image-type";
 export { ImageHelper, IImageCompareOptions } from "./lib/image-helper";
+export { DeviceOrientaion } from "./lib/enums/device-orientatioin";
```

Comment:
Typo in orientatio**_i_**n

Reply:
Also typo in `export { DeviceOrientaion }`